docs/book/src/clusterctl/configuration.md

```diff
@@ -29,6 +29,14 @@ providers:
   - name: "cluster-api"
     url: "https://github.com/myorg/myforkofclusterapi/releases/latest/core-components.yaml"
     type: "CoreProvider"
+  # add a custom provider on a self-host GitLab
+  - name: "my-infra-provider"
+    url: "https://gitlab.example.com/api/v4/projects/myorg%2Fmyrepo/packages/generic/myrepo/v1.2.3/infrastructure-components.yaml"
+    type: "InfrastructureProvider"
+  # override a pre-defined provider on a self-host GitLab
+  - name: "cluster-api"
+    url: "https://gitlab.example.com/api/v4/projects/external-packages%2Fcluster-api/packages/generic/cluster-api/v1.1.3/core-components.yaml"
+    type: "CoreProvider"
```

Comment:
Can we drop this example, we already have the previous one.
(there is already another override for the same provider in L29, so I fear having two overrides for the same provider could be confusing for end-users)